lib/scrub/autoscrub.rb

```diff
@@ -26,12 +26,16 @@ def initialize(path)
       @item_type = @member_holding_file.item_type_from_filename
       @output_dir = @output_struct.date_subdir!("output")
       @log_dir = @output_struct.date_subdir!("log")
+      @out_files = []
 
       # Once we have @member_id and @item_type,
       # build a log path and re-register the service logger to log to that path
-      logger_path = File.join(@log_dir, "#{@member_id}_#{@item_type}.log")
+      ymd = Time.new.strftime("%F")
+      @logger_path = File.join(@log_dir, "#{@member_id}_#{@item_type}_#{ymd}.log")
+      puts "autoscrub logging to #{@logger_path}"
```

Comment:
Extraneous/duplicate -- we're logging it via `Services.logger` at line 48 already

Reply:
Line 48 is `Services.logger` ($stderr) saying that we logging some specifics to the file `@logger_path`. These are 2 separate logs. Could perhaps be done clearer, but it is intentional.

Reply:
I'm not sure I understand, but regardless we shouldn't be using `puts`

Reply:
And now we aren't, so everything is fine